README.md

```diff
@@ -27,10 +27,20 @@ cordova plugin add https://github.com/samzilverberg/cordova-mixpanel-plugin.git
 **window.mixpanel.people:**
 
 - identify(distinctId, onSuccess, onFail)
-- registerPushToken(apnsOrGcmToken, onSuccess, onFail)
+- registerPushId(registrationId, onSuccess, onFail)
```

Comment:
i have a new suggestion: setPushNotificationsToken.
seems to me it is the most obvious self documenting name for this.
lemme know what ya think